src/Strategy.sol

```diff
@@ -45,14 +44,15 @@ contract Strategy is BaseHealthCheck, IUniswapV3SwapCallback, AuctionSwapper {
     address public constant AJNA_TOKEN =
         0x9a96ec9B57Fb64FbC60B423d1f4da7691Bd35079;
 
+    // review: summerfi is upgradable, right? We are doomed if they break their api?
```

Comment:
[ProxyActions](https://etherscan.io/address/0x3637df43f938b05a71bb828f13d9f14498e6883c#writeContract) is not a proxy, so we should be fine.